.github/workflows/test.yml

```diff
@@ -172,7 +172,7 @@ jobs:
     uses: ./.github/workflows/docker-build.yml
     secrets: inherit
     with:
-      push: false
+      push: true
```

Comment:
This pushes the image.

I am somewhat confused. I triggered a test build manually on `main` earlier today. It does not include this change and the image has [still been uploaded](https://github.com/agglayer/agglayer/pkgs/container/agglayer/382120050?tag=main) for some reason.

Reply:
Turns out there is a separate step to push the image but it is not triggering for `main` merge, only for manual run. Testing a fix now.